src/provider/index.ts

```diff
@@ -126,6 +126,13 @@ export class Main {
     private async getChannelMembers(payload: GetChannelMembersPayload, source: ProviderIdentity): Promise<Identity[]> {
         return this._contexts.getChannelMembers(payload, source);
     }
+
+    private async intentListenerReady(payload: IntentListenerReadyPayload, identity: ProviderIdentity): Promise<void> {
+        const app = this._model.getWindow(identity);
+        app!.intents[payload.intent] = true;
```

Comment:
Move ! to end of line 131?

Reply:
I'm adding a guard for `app`, throwing an Error if it's not there, in the next PR I'm working on.